.github/ISSUE_TEMPLATE/09-component-issue.yml

```diff
@@ -0,0 +1,18 @@
+name: Report an issue on a component
+description: Report any issues regarding a component on GOV.UK
+title: "[Enter the name of the component]: [Enter the issue with component]"
+labels: ["component"]
+projects: ["alphagov/100"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Steps to creating an issue for a GOV.UK component
+        - Update the title according to the proposed format
+        - In the textarea below, provide a detailed description of the issue
```

Comment:
Correct textarea to text area when it's being referred to as an instruction, and same for other templates.